Modules/_testcapi/func_events.c

```diff
@@ -0,0 +1,168 @@
+#include "parts.h"
+
+#define NUM_WATCHERS 2
+static PyObject *pyfunc_watchers[NUM_WATCHERS];
+static int watcher_ids[NUM_WATCHERS] = {-1, -1};
+
+static PyObject *get_id(PyObject *obj) {
+    PyObject *builtins = PyEval_GetBuiltins();
```

Comment:
Please add a comment that this is a borrowed reference.

```suggestion
    PyObject *builtins = PyEval_GetBuiltins();  // borrowed ref.
```